templates/explore/user_list.tmpl

```diff
@@ -0,0 +1,31 @@
+<div class="flex-list">
+	{{range .Users}}
+		<div class="flex-item flex-item-center">
+			<div class="flex-item-leading">
+				{{ctx.AvatarUtils.Avatar . 48}}
+			</div>
+			<div class="flex-item-main">
+				<div class="flex-item-title">
+					{{template "shared/user/name" .}}
+					{{if .Visibility.IsPrivate}}
+						<span class="ui basic tiny label">{{ctx.Locale.Tr "repo.desc.private"}}</span>
+					{{end}}
+				</div>
+				<div class="flex-item-body">
+					{{if .Location}}
+						<span class="flex-text-inline">{{svg "octicon-location"}}{{.Location}}</span>
+					{{end}}
+					{{if and $.ShowUserEmail .Email $.IsSigned (not .KeepEmailPrivate)}}
```

Comment:
I can see this template is reused, but would it cause over-abstraction?

For example, when the users are listed on the admin page, I guess it should always show user's email?